src/main/index.ts

```diff
@@ -1,58 +1,42 @@
-import { BrowserWindow, app } from "electron";
+import "reflect-metadata";
+import { Container, injectable } from "inversify";
+import entities from "./entities";
+import AppService from "./services/app.service";
 
-let mainWindow: BrowserWindow | null;
-const winURL =
-  process.env.NODE_ENV === "development"
-    ? `http://localhost:9080`
-    : `file://${__dirname}/index.html`;
-
-function createWindow() {
-  /**
-   * Initial window options
-   */
-  mainWindow = new BrowserWindow({
-    height: 563,
-    useContentSize: true,
-    width: 1000
-  });
-
-  mainWindow.loadURL(winURL);
+/**
+ * Create the DI Container
+ *
+ * @type {Container}
```

Comment:
I don't think JSDoc is necessary here. A plain comment is more lightweight and the type comment is redundant with the code:

```
// Create a container for dependency injection.
let container = new Container();
```